### PR TITLE
Updates to Embedded Authentication > SDK Use Cases: Sign Out and Password and Email

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Improved images and sequence flows for sign out and email password use cases, as well as minor text tweaks and image alt-text additions.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-408181](https://oktainc.atlassian.net/browse/OKTA-408181)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ The first step is to create a link, button, or other similar UI element that all
 
 <div class="common-image-format">
 
-![Sign out link](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-link.png
- "Sign out link")
+![Displays a screenshot of a logout button that signs a user out of the application.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-link-nodejs.png)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md:9]:
```suggestion
![Displays a screenshot of a Logout button that signs a user out of the application.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-link-nodejs.png)
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md:9]: Capitalize to be consistent with the other alt-text in the topic that refer to buttons?

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md [hunk 2]
```diff
@@ -4,45 +4,43 @@ Build a sign-in page that captures both the user’s name and password, as shown
...
 
 </div>
 
 When the user initiates the sign-in process, your app needs to create a new `OktaAuth` object and set its `username` and `password` properties to the values entered by the user. Send this object to the [`idx.authenticate`](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate) method to start the authentication process. This call begins the multi-factor authentication flow.
 
-If the username and password are valid, `idx.authenticate` should return a status of `Idx.Status:PENDING`. This status indicates that an additional factor needs to be verified before the sign-in. In addition to the status, the `nextStep` field that is included in the response identifies the input parameters of the next step, which is an authenticator of type email in this case:
+If the username and password are valid, `idx.authenticate` returns a status of `Idx.Status:PENDING`. This status indicates that an additional factor needs to be verified before the sign in. In addition to the status, the `nextStep` field that is included in the response identifies the input parameters of the next step, which is an authenticator of type email in this case:
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:13]:
```suggestion
If the username and password are valid, `idx.authenticate` returns a status of `Idx.Status:PENDING`. This status indicates that an additional factor needs to be verified before the sign in. In addition to the status, the `nextStep` field that is included in the response identifies the input parameters of the next step, which in this case is for the email authenticator:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md [hunk 3]
```diff
@@ -53,12 +51,11 @@ If not already done, build the email verification code page that accepts the cod
...
+The next step is to call `idx.authenticate` and pass in the email verification code `({ verificationCode: '1213' })` and handle the response from the call. If the email code is valid, the method returns a status of `Idx.Status:SUCCESS` and tokens. This status signifies that there are no more factors that are waiting to be enrolled and verified. If the steps in [Set up your Okta org (for multi-factor use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-multifactor-use-cases) were properly followed, the user has successfully signed in and should be sent to the default sign-in home page.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:58]:
```suggestion
The next step is to call `idx.authenticate` and pass in the email verification code (for example,`({ verificationCode: '1213' })`) and handle the response from the call. If the email code is valid, the method returns a status of `Idx.Status:SUCCESS` and tokens. This status signifies that there are no more factors waiting to be enrolled and verified. If the steps in [Set up your Okta org (for multi-factor use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-multifactor-use-cases) were properly followed, the user has successfully signed in and should be sent to the default sign-in home page.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/summaryofsteps.md [hunk 4]
```diff
@@ -4,9 +4,6 @@ The following shows the sequence of steps for the sign-in with password and emai
 
 <div class="common-image-format">
 
-![Sign in with email and password](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-email-nodejs.png
- "Sign in with email and password")
+![Displays a flow diagrams for the sign in with email and password use case.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-email-nodejs.png)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/summaryofsteps.md:7]:
```suggestion
![Displays a flow diagram for the sign in with email and password use case.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-email-nodejs.png)
```

**Review by vanngo-okta — APPROVED** [review]
Looks good! A few small nits, but I'm approving.
Also if you have time, ensure the command phrases in the sequence diagrams are consistent ("Returns response" vs. "Return response; "Clicks ..." vs. "Click xxx") and avoid using "e.g.".

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/nodejs/integrationsteps.md:58]: weird, I thought I change those things. (But probably missed them.) Thanks.

**Comment by brianduffield-okta** [comment]
Thanks, Van. I'll address the command phrases in my next "update" branch.

**Pull request merged**